.husky/pre-commit

```diff
@@ -13,4 +13,4 @@ fi
 if [ -n "$(git status packages/backend --porcelain --untracked-files=no)" ]; then
   cd packages/backend
   npx lint-staged
-fi
+fiq
```

Comment:
**issue (blocking):** this seems to be a syntax error (fiq instead of fi), causing all related tests to fail.